packages/parrot-fetch/src/ParrotFetch.js

```diff
@@ -37,9 +43,14 @@ class ParrotFetch extends Parrot {
       return this.contextFetch(input, init);
     }
     const { body, status } = response;
-    return Promise.resolve(
-      new Response(new Blob([JSON.stringify(body)], { status, type: 'application/json' }))
-    );
+    const responseBlob = new Blob([JSON.stringify(body)]);
```

Comment:
I think this should still set the MIME type,  I suspect that was the original intention and that status was incorrectly included. 

```suggestion
    const responseBlob = new Blob([JSON.stringify(body)], { type: type: 'application/json' });
```

Reply:
thanks! Forgot that blobs also have a mime type associated